web/src/components/nav_bar/messages.js

```diff
@@ -0,0 +1,5 @@
+const messages = {
+  logoQuip: "West coast swing for all",
```

Comment:
Maybe title case for WCS?

```suggestion
  logoQuip: "West Coast Swing for all",
```

Reply:
Nah prefer non-title case so its more conversational (stole this from work). Also updated it to be Connect. Create. Dance.